portfolio/src/main/webapp/style.css

```diff
@@ -10,4 +10,33 @@
 
 .invisible {
     display: none;
-}
\ No newline at end of file
+}
+
+.topnav {
+  background-color: rgb(0, 0, 0);
+  overflow: hidden;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: center;
+  color: #ffffff;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 18px;
+  margin:10px;
```

Comment:
nit: I think you're missing a space between the ":" and "10px". Not a big deal, just want to be consistent with the rest of the file